src/table/virtualizedDataTable.jsx

```diff
@@ -140,48 +498,90 @@ class VirtualizedDataTable extends Component {
         },
       });
     };
-  }
 
   // handles the resizable column drag end
-  prvHandleResizeDragStop(evt) {
+  prvHandleResizeDragStop = (evt) => {
+    const { onColumnResizeEndCallback } = this.props;
     const { resizeDrag: { dragColumn, dragStart, origWidth } } = this.state;
     const newColumnWidth = origWidth + (evt.clientX - dragStart);
     if (newColumnWidth > 0) {
-      this.props.onColumnResizeEndCallback(newColumnWidth, dragColumn);
+      onColumnResizeEndCallback(newColumnWidth, dragColumn);
     }
 
+    this.prvRecomputeTables();
+
     this.setState({
       resizeDrag: null,
       recentDragEnd: dragColumn,
     });
-  }
+  };
+
+  // handles mouse wheel.
+  prvHandleWheel = (evt) => {
+    if (this.prvCellGridDOM) {
+      evt.preventDefault();
+
+      const { wheelDelta } = this.props;
+      const { state: { scrollTop, scrollLeft } } = this.prvCellGridRef;
+      const delta = evt.deltaY || evt.detail || evt.wheelDelta;
+      const newScrollTop = Math.max(0, scrollTop + ((delta > 0) ? wheelDelta : -wheelDelta));
+      this.prvCellGridDOM.scrollTop = newScrollTop;
+
+      this.prvCellGridRef.handleScrollEvent({
+        scrollLeft,
+        scrollTop: newScrollTop,
+      });
+    }
+  };
 
   // handles scrolling in the Grid
   // TODO: we are compensating for a bug in Chrome or React where issuing a React update cycle
   // (setState) under a scroll event can cause laggy scroll performance, and temporary misalignment
   // of the grids.  Once Chrome or React fixes this issue, we should switch to use MultiGrid and
   // stop directly setting transforms here
-  prvHandleScroll({ scrollLeft }) {
+  prvHandleScroll = ({ scrollLeft, scrollTop }) => {
     try {
-      const transformStyle = `translate(-${scrollLeft}px, 0px)`;
-      const groupHeader = ReactDOM.findDOMNode(this.prvGroupHeader); // eslint-disable-line max-len, react/no-find-dom-node
-      if (groupHeader) {
-        groupHeader.style.transform = transformStyle;
-      }
-      const headerGrid = ReactDOM.findDOMNode(this.prvHeaderGrid); // eslint-disable-line max-len, react/no-find-dom-node
-      if (headerGrid) {
-        headerGrid.style.transform = transformStyle;
-      } else if (this.prvHeaderGrid && this.prvHeaderGrid._scrollingContainer) { // eslint-disable-line max-len, no-underscore-dangle
-        this.prvHeaderGrid._scrollingContainer.style.transform = transformStyle; // eslint-disable-line max-len, no-underscore-dangle
-      }
```

Comment:
Handle left is the legacy behavior, which kept the header table in sync with the grid table